test/cpp/tensorexpr/test_loopnest.cpp

```diff
@@ -1892,6 +1892,136 @@ TEST(LoopNest, LoopNestComputeAt_3) {
   }
 }
 
+using Axis = const VarHandle&;
+
+void checkIR(Stmt* s, const std::string& pattern) {
+  std::ostringstream oss;
+  oss << *s;
+  torch::jit::testing::FileCheck().run(pattern, oss.str());
+}
```

Comment:
Nit: since you're adding this, it would be good to use it in existing tests too.

Reply:
Yep, will update things in a follow up diff :-)